### DISCUSSION TITLE
Please help my friend!

### DISCUSSION THREAD

**Discussion by ch10874** [community/community#137595] [New to GitHub] [answered]
Hi
My friend is an old dev and he has his github account.
But unfortunately, he don't remember his previous phone number(phone number two step verification). So he can't log in and use his email now.
When log in his github, it requires verification code, but he can't see his mail box.
What is the solution?
He has necessary private repositories there.
Please help my friend!

**Comment by AidenJamesYt** [comment, marked as answer]
Hi!

Here is a link to [Github's Support Website](https://support.github.com/).

Here is a link to open a [Support Ticket](https://support.github.com/request).

If you need further assistance, let me know! :D

**Reply by ch10874** [reply]
Thank you!

**Reply by AidenJamesYt** [reply]
no problem! :D

**Reply by ghostinhershell** [reply]
Thanks for hopping in and lending a hand here @AidenJamesYt! 🚀

**Comment by JBarrera26** [comment]
hi, i´ve tried to get github education for student and i have submit my id card, Schedule and univerity´s certificate but I still getting reject, any advice?

**Reply by ipankervank** [reply]
Ok

**Reply by AidenJamesYt** [reply]
> hi, i´ve tried to get github education for student and i have submit my id card, Schedule and univerity´s certificate but I still getting reject, any advice?

unrelated question, but you can use the same stuff I gave @ipankervank 

Here is a link to [Github's Support Website](https://support.github.com/).

Here is a link to open a [Support Ticket](https://support.github.com/request).

**Comment by ch10874** [comment]
Okay.
Good luck, my friends.
